code/build/template/include/extensions/[+python orxPy.h]

```diff
@@ -0,0 +1,2613 @@
+  {                                                    \
+    target = 0;                                        \
+  }
+
+  void orxPy_InitCallbacks()
+  {
+    // Get orx module
+    py_GlobalRef pyMod = py_getmodule("orx");
+    py_Name pyName;
+    py_ItemRef pyCallback = orxNULL;
+
+    // Set module reference
+    sstPy.pyMod = pyMod;
+
+    // Get function names for functions which are defined
+    SET_CALLBACK(sstPy.pyInit, orxPY_KZ_DEFAULT_INIT)
+    SET_CALLBACK(sstPy.pyUpdate, orxPY_KZ_DEFAULT_UPDATE)
+    SET_CALLBACK(sstPy.pyExit, orxPY_KZ_DEFAULT_EXIT)
+    SET_CALLBACK(sstPy.pyOnCreate, orxPY_KZ_DEFAULT_OBJECT_CREATE_EVENT_HANDLER)
+    SET_CALLBACK(sstPy.pyOnDelete, orxPY_KZ_DEFAULT_OBJECT_DELETE_EVENT_HANDLER)
+    SET_CALLBACK(sstPy.pyOnCollide, orxPY_KZ_DEFAULT_PHYSICS_COLLIDE_EVENT_HANDLER)
+    SET_CALLBACK(sstPy.pyOnSeparate, orxPY_KZ_DEFAULT_PHYSICS_SEPARATE_EVENT_HANDLER)
+    SET_CALLBACK(sstPy.pyOnShaderParam, orxPY_KZ_DEFAULT_SHADER_PARAM_EVENT_HANDLER)
+
+    // Register commands
+    orxCOMMAND_REGISTER(orxPY_KZ_COMMAND_EXEC, orxPy_CommandPyExec, "Success?", orxCOMMAND_VAR_TYPE_BOOL, 1, 1, {"Source", orxCOMMAND_VAR_TYPE_STRING}, {"HideException? = false", orxCOMMAND_VAR_TYPE_BOOL});
+
+    // Register event handler for events with handlers
+    orxU32 u32ObjectAddFlags = orxEVENT_KU32_FLAG_ID_NONE;
+    u32ObjectAddFlags |= sstPy.pyOnCreate != 0 ? orxEVENT_GET_FLAG(orxOBJECT_EVENT_CREATE) : orxEVENT_KU32_FLAG_ID_NONE;
+    u32ObjectAddFlags |= sstPy.pyOnDelete != 0 ? orxEVENT_GET_FLAG(orxOBJECT_EVENT_DELETE) : orxEVENT_KU32_FLAG_ID_NONE;
+    if (u32ObjectAddFlags != orxEVENT_KU32_FLAG_ID_NONE)
+    {
+      orxEvent_AddHandler(orxEVENT_TYPE_OBJECT, orxPy_EventHandler);
+      orxEvent_SetHandlerIDFlags(orxPy_EventHandler, orxEVENT_TYPE_OBJECT, orxNULL, u32ObjectAddFlags, orxEVENT_KU32_MASK_ID_ALL);
+    }
+    orxU32 u32PhysicsAddFlags = orxEVENT_KU32_FLAG_ID_NONE;
+    u32PhysicsAddFlags |= sstPy.pyOnCollide != 0 ? orxEVENT_GET_FLAG(orxPHYSICS_EVENT_CONTACT_ADD) : orxEVENT_KU32_FLAG_ID_NONE;
+    u32PhysicsAddFlags |= sstPy.pyOnSeparate != 0 ? orxEVENT_GET_FLAG(orxPHYSICS_EVENT_CONTACT_REMOVE) : orxEVENT_KU32_FLAG_ID_NONE;
+    if (u32PhysicsAddFlags != orxEVENT_KU32_FLAG_ID_NONE)
+    {
+      orxEvent_AddHandler(orxEVENT_TYPE_PHYSICS, orxPy_EventHandler);
+      orxEvent_SetHandlerIDFlags(orxPy_EventHandler, orxEVENT_TYPE_PHYSICS, orxNULL, u32PhysicsAddFlags, orxEVENT_KU32_MASK_ID_ALL);
+    }
+    orxU32 u32ShaderAddFlags = orxEVENT_KU32_FLAG_ID_NONE;
+    u32ShaderAddFlags |= sstPy.pyOnShaderParam != 0 ? orxEVENT_GET_FLAG(orxSHADER_EVENT_SET_PARAM) : orxEVENT_KU32_FLAG_ID_NONE;
+    if (u32ShaderAddFlags != orxEVENT_KU32_FLAG_ID_NONE)
+    {
+      orxEvent_AddHandler(orxEVENT_TYPE_SHADER, orxPy_EventHandler);
+      orxEvent_SetHandlerIDFlags(orxPy_EventHandler, orxEVENT_TYPE_SHADER, orxNULL, u32ShaderAddFlags, orxEVENT_KU32_MASK_ID_ALL);
+    }
+  }
+
+#undef SET_CALLBACK
+
+  orxSTATUS orxPy_Init()

```

Comment:
This calls the user's/game's `orx.init` function, called from the init project's main source file.